ch.protonmail.protonmail-import-export-app.yml

```diff
@@ -0,0 +1,33 @@
+id: ch.protonmail.protonmail-import-export-app
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: "20.08"
+command: protonmail-import-export-app
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network
+  - --device=dri
+  - --talk-name=org.freedesktop.secrets
+modules:
+  - libsecret.json
+  - name: protonmail-import-export-app
+    buildsystem: simple
+    build-commands:
+      - install -Dp -m 644 ${FLATPAK_ID}.metainfo.xml /app/share/metainfo/${FLATPAK_ID}.metainfo.xml
+      - ar x protonmail-import-export-app.deb
+      - tar -xf data.tar.xz
+      - mv usr/lib/* /app/lib/
+      - install -Dp -m 644 usr/share/icons/hicolor/scalable/apps/protonmail-import-export-app.svg /app/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.svg
+      - install -Dp -m 644 usr/share/applications/protonmail-import-export-app.desktop /app/share/applications/${FLATPAK_ID}.desktop
+      - desktop-file-edit --set-key="Icon" --set-value=${FLATPAK_ID} /app/share/applications/${FLATPAK_ID}.desktop
+      - desktop-file-edit --set-key="Categories" --set-value="Office;Email" /app/share/applications/${FLATPAK_ID}.desktop
+      - ln -s /app/lib/protonmail/import-export/protonmail-import-export-app /app/bin/
+    sources:
+      - type: file
```

Comment:
If there's a deb repo, you should add flatpak-external-data-checker metadata here.

Reply:
It is not a repo. But added metadata anyways.